### PR TITLE
Update Forsakringskassan.se.xml

### PR DESCRIPTION
**Pull request by I-have-50-open-PRs-to-HTTPS-EVERYWHERE**

### CODE REVIEW

**Comment by J0WI** [comment]
There are just a few subdomains left. Could you please check them?

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@J0WI Added.

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@J0WI Added.

File: src/chrome/content/rules/Forsakringskassan.se.xml [hunk 1]
```diff
@@ -1,6 +1,22 @@
+<!--
+	Timeout:
+		forslag.forsakringskassan.se
+		separeradeforaldrar.forsakringskassan.se
+		separationspodden.forsakringskassan.se
+	Invalid Certificate:
+		lagrummet.forsakringskassan.se
+		kassakollen.forsakringskassan.se
```
Comment (J0WI) [inline src/chrome/content/rules/Forsakringskassan.se.xml]: Just a chain issue

File: src/chrome/content/rules/Forsakringskassan.se.xml [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+	Unable to Connect:
+		mobil.forsakringskassan.se
```
Comment (J0WI) [inline src/chrome/content/rules/Forsakringskassan.se.xml]: Does not exist

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@J0WI Fixed.

**Pull request merged**